### ISSUE TITLE
Cannot download thirdparty archives when build

### ISSUE THREAD

**Issue by ykPrograWorld** [apache/incubator-doris#1102]
![图片](https://user-images.githubusercontent.com/26919124/57268253-b172e800-70b5-11e9-8d28-3d36d31b29a1.png)
Why are third-party package addresses always invalid？？？

**Comment by imay** [comment]
@ykPrograWorld 

This issue has been fixed, #914 , 

You can update code to master or cherry-pick #916 patch to fix it

**Comment by ykPrograWorld** [comment]
> 
> 
> @ykPrograWorld
> 
> This issue has been fixed, #914 ,
> 
> You can update code to master or cherry-pick #916 patch to fix it

The package I downloaded is from this website http://59.80.44.44/www.apache.org/dyn/closer.cgi/incubator/doris/0.9.0-incubating/apache-doris-0.9.0-incubating-src.tar.gz
Isn't this the latest package?
Have you solved the problem in your way anyway
Thanks

**Comment by imay** [comment]
We have fixed this problem in master, and the new version has not been released.

So, if you want to fix it, you can patch #916 to our release code

**Comment by ykPrograWorld** [comment]
> 
> 
> We have fixed this problem in master, and the new version has not been released.
> 
> So, if you want to fix it, you can patch #916 to our release code

where is the release code?

**Comment by imay** [comment]
> > We have fixed this problem in master, and the new version has not been released.
> > So, if you want to fix it, you can patch #916 to our release code
> 
> where is the release code?

the code you have downloaded.

http://59.80.44.44/www.apache.org/dyn/closer.cgi/incubator/doris/0.9.0-incubating/apache-doris-0.9.0-incubating-src.tar.gz

**Comment by ykPrograWorld** [comment]
> 
> 
> > > We have fixed this problem in master, and the new version has not been released.
> > > So, if you want to fix it, you can patch #916 to our release code
> > 
> > 
> > where is the release code?
> 
> the code you have downloaded.
> 
> http://59.80.44.44/www.apache.org/dyn/closer.cgi/incubator/doris/0.9.0-incubating/apache-doris-0.9.0-incubating-src.tar.gz

I can only change my local package, there is no way to change package on the web site.

**Comment by imay** [comment]
> > > > We have fixed this problem in master, and the new version has not been released.
> > > > So, if you want to fix it, you can patch #916 to our release code
> > > 
> > > 
> > > where is the release code?
> > 
> > 
> > the code you have downloaded.
> > http://59.80.44.44/www.apache.org/dyn/closer.cgi/incubator/doris/0.9.0-incubating/apache-doris-0.9.0-incubating-src.tar.gz
> 
> I can only change my local package, there is no way to change package on the web site.

Yes, change your local package, 

modify `thirdparty/vars.sh` file 

change 

```
LIBEVENT_DOWNLOAD="http://palo-opensource.gz.bcebos.com/libevent-20180622-24236aed01798303745470e6c498bf606e88724a.zip?authorization=bce-auth-v1%2F069fc2786e464e63a5f1183824ddb522%2F2018-06-22T02%3A37%3A48Z%2F-1%2Fhost%2F031b0cc42ab83ca4e0ec3608cba963e95c1ddc46fc70a14457323e2d7960e6ef"
```
to

```
LIBEVENT_DOWNLOAD="https://doris-incubating-repo.bj.bcebos.com/thirdparty/libevent-20180622-24236aed01798303745470e6c498bf606e88724a.zip"
```

**Comment by ykPrograWorld** [comment]
> 
> 
> > > > > We have fixed this problem in master, and the new version has not been released.
> > > > > So, if you want to fix it, you can patch #916 to our release code
> > > > 
> > > > 
> > > > where is the release code?
> > > 
> > > 
> > > the code you have downloaded.
> > > http://59.80.44.44/www.apache.org/dyn/closer.cgi/incubator/doris/0.9.0-incubating/apache-doris-0.9.0-incubating-src.tar.gz
> > 
> > 
> > I can only change my local package, there is no way to change package on the web site.
> 
> Yes, change your local package,
> 
> modify `thirdparty/vars.sh` file
> 
> change
> 
> ```
> LIBEVENT_DOWNLOAD="http://palo-opensource.gz.bcebos.com/libevent-20180622-24236aed01798303745470e6c498bf606e88724a.zip?authorization=bce-auth-v1%2F069fc2786e464e63a5f1183824ddb522%2F2018-06-22T02%3A37%3A48Z%2F-1%2Fhost%2F031b0cc42ab83ca4e0ec3608cba963e95c1ddc46fc70a14457323e2d7960e6ef"
> ```
> 
> to
> 
> ```
> LIBEVENT_DOWNLOAD="https://doris-incubating-repo.bj.bcebos.com/thirdparty/libevent-20180622-24236aed01798303745470e6c498bf606e88724a.zip"
> ```

I've done that and my problem have been solved
Thanks
